gpustat.py

```diff
@@ -492,7 +497,17 @@ def main():
                         version=('gpustat %s' % __version__))
     args = parser.parse_args()
 
-    print_gpustat(**vars(args))
+    if args.interval > 0:
+        term = Terminal()
+        with term.fullscreen():
+            while 1:
+                try:
+                    print_gpustat(clear_term=True, **vars(args))
+                    time.sleep(args.interval)
```

Comment:
Why don't we simply clear terminal here, rather than passing parameters everywhere?

Reply:
Yes, it was in my first commit 800dbf9 . But the data query in some cases takes a long time and the screen starts blinking.

Reply:
I see. It makes sense.